viadot/utils.py

```diff
@@ -345,3 +351,31 @@ def union_dict(*dicts):
 
     """
     return dict(chain.from_iterable(dct.items() for dct in dicts))
+
+
+def handle_if_empty_file(if_empty: Literal["warn", "skip", "fail"] = "warn"):
+    if if_empty == "warn":
+        logger.warning("The input file is empty.")
+    elif if_empty == "skip":
+        raise SKIP()
```

Comment:
This could be improved to pass a useful message, eg. by accepting a param `message` which you pass to the warnings/exceptions. Then in `check_if_empty_file()` you can pass the path to the file inside the `message`.